Apps/Src/Fans.c

```diff
@@ -8,31 +8,26 @@ Uses Pins PC6,7 and PB14,15
 #include "Fans.h"
 #include "Contactor.h"
 
-/**
- * @brief   Initialize all the GPIO pins connected to each Fan
- * @param   None
- * @return  None
- */
 void Fans_Init(void){
-    #ifdef SIMULATION
-        Simulator_Log(LOG_INFO, "Fans Initialized\n");
-    #endif
-    BSP_PWM_Init();
+    Fans_SetAll(4); //start with all fans half speed
```

Comment:
``Fans_Init()`` should not be setting the fans. I don't think anything here should have changed at all.